### ISSUE TITLE
Sensor status - unknown

### ISSUE THREAD

**Issue by rafkan20** [YorkshireIoT/ha-google-fit#40]
### Checklist

- [X] I have filled out the template to the best of my ability.
- [X] This only contains 1 feature request (if you have multiple feature requests, open one feature request for each feature request).
- [X] This issue is not a duplicate feature request of [previous feature requests](https://github.com/YorkshireIoT/ha-google-fit/issues?q=is%3Aissue+label%3A%22Feature+Request%22+).

### Is your feature request related to a problem? Please describe.

Until the integration version v0.2.0-b2, sensors after a daily reset had a value of 0.
From version v0.2.0-b3, after a daily reset, the sensors assume the "unknown" state.

![jazz](https://images92.fotosik.pl/660/e8ac68c3f5c45a43med.jpg)

It doesn't look good on the panel.

Is it possible to return the sensors to 0 after the daily reset?

### Describe the solution you'd like

Home Assistant 2023.4.6
Supervisor 2023.04.0
Operating System 10.0
Interfejs użytkownika: 20230411.1 - latest

### Describe alternatives you've considered

N/A

### Additional context

N/A

**Comment by YorkshireIoT** [comment]
Hi @rafkan20 ,

This was introduced deliberately when the sleep sensors were added. I did this because things like REM, light + deep sleep aren't always defined and so a sensor value of "Unknown" made most sense.

The same actually applies to the steps and other sensors as well, in that if there is no data there then the value of the sensor is unknown.

I have three options for this and I'd be interested in your opinion:
1. Take everything back to 0, which could be confusing when the sensor type isn't available in your account but it does make more sense to the average user.
2. Take everything back to using 0 except the sleep sensor types. This isn't very expandable and does then have inconsistency and code bloat issues
3. Allow the user to choose in the integration configuration whether they want to use 0 or unknown as the default sensor value.
